.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
it's weird there are comments in this json file (rendering it invalid), did you add those manually or?

Reply:
That comment was generated by VS Code.

Reply:
I will delete it.